test/passing/module_item_spacing.ml.ref

```diff
@@ -0,0 +1,69 @@
+[@@@ocamlformat "module-item-spacing=compact"]
+
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+let f x = x + 1
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+let g = ()
+
+let f = function
+  | `a | `b | `c -> foo
+  | `xxxxxxxxxxxxxxxxxx ->
+      yyyyyyyyyyyyyyyyyyyyyyyy
+        zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+        zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+        zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+
+let x = 1
+let y = 2
+let z = this one is pretty looooooooooooooooooooooooooooooooooong
+let z = so is this oooooooooooooooooooooooooooooooooooooooooooone
+
+module A = AA
+module B = BB
+open AA
+module C = CC
+
+module M =
+  X
+    (Y)
+    (struct
+      let x = k
+    end)
```

Comment:
is there a way to turn this into a one-liner?

Reply:
This breaks because structures are force-broken. This could be made configurable, with a 2-line change in the `Pmod_structure` case of `fmt_module_expr` to put `@<1000 N>` instead of `@<1 N>` only when the config option is set to sparse.

Reply:
I basically have the implementation already, I can send a PR later today.